src/backend/utils.ts

```diff
@@ -256,9 +257,12 @@ async function handleExit() {
         logInfo([`Unable to kill ${procName}, ignoring.`, error])
       }
     })
+  }
+  // Kill all child processes, closing browser and native games that are running
+  callAllAbortControllers()
```

Comment:
this has to be called before the cleanUp function so that that launch handlers can finish executing